controllers/metering/main.go

```diff
@@ -90,6 +91,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Metering")
 		os.Exit(1)
 	}
+	if err = (&controllers.PodResourceReconciler{
```

Comment:
Maybe pod resource controller can split into self-hosting controllers?

Reply:
The next iteration will change to facilitate decoupling